### PR TITLE
feat(certificate) Use cert-manager.io/v1

### PR DESCRIPTION
**Pull request by holyhope**
Signed-off-by: Pierre Péronnet <pierre.peronnet@ovhcloud.com>

### CODE REVIEW

**Review by bitsf — APPROVED** [review]

File: apis/goharbor.io/v1alpha2/notaryserver_types.go [hunk 1]
```diff
@@ -80,8 +80,8 @@ type NotaryServerTrustServiceRemoteSpec struct {
 	Port int64 `json:"port,omitempty"`
 
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=ecdsa
-	KeyAlgorithm certv1.KeyAlgorithm `json:"keyAlgorithm,omitempty"`
+	// +kubebuilder:default=ECDSA
```
Comment (bitsf) [inline apis/goharbor.io/v1alpha2/notaryserver_types.go:83]: CI seems failed because of this

**Review by bitsf — CHANGES_REQUESTED** [review]
ci failed

**Comment by steven-zou** [comment]
@holyhope 

Fix CI failures?

**Comment by steven-zou** [comment]
@holyhope 

Any progress of fixing this PR?

**Comment by steven-zou** [comment]
Fixing by PR https://github.com/goharbor/harbor-operator/pull/502

**Pull request closed without merging**